source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -140,16 +140,21 @@ Consistency
 ~~~~~~~~~~~
 
 ``mongosync`` supports :term:`eventual consistency` on the destination
-cluster. Read Consistency is not guaranteed on the destination cluster,
-the source and destination clusters may differ at a given point in
-time. However, if new writes are halted on the source cluster, the
-source and destination clusters will eventually hold the same data.
+cluster. Read consistency is not guaranteed on the destination cluster until 
+commit. Before committing, the source and destination clusters may differ at a 
+given point in time. However, if new writes are halted on the source cluster, 
+the source and destination clusters will eventually hold the same data.
+
+.. warning:: 
+   
+     We do not recommend using ``mongosync`` for disaster recovery. To learn 
```

Comment:
I would recommend removing this. Customers will decide for themselves whether running mongosync continuously makes sense for their purposes, given the risks. Also, right now customers are encountering this page because they are searching for "disaster recovery mongodb", when the go to solution should actually be replication with automated failover in many cases. I'd like to exclude the string "disaster recovery" from this page.

Reply:
Gotcha. Updating.

Reply:
In that case, should we be explicitly mentioning that they should use replication w/ automated failover in the case of a disaster scenario? With us deleting the disaster recovery page, we never really provide a solution for it.

Reply:
We don't really have a solution for it